pkg/cloud/ovirt/machine/actuator.go

```diff
@@ -338,7 +338,8 @@ func (actuator *OvirtActuator) reconcileProviderStatus(machine *machinev1.Machin
 
 func (actuator *OvirtActuator) reconcileProviderID(machine *machinev1.Machine, instance *clients.Instance) {
 	id := instance.MustId()
-	machine.Spec.ProviderID = &id
+	providerID := ovirt.ProviderIDPrefix + id
```

Comment:
is providerID variable needed?

Reply:
Maybe that is some golang knowledge I'm missing,
Is there a way to give a pointer to a calculated object?